backend/amd64/arch.mli

```diff
@@ -0,0 +1,81 @@
+# 2 "asmcomp/amd64/arch.mli"
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           *)
+(*                                                                        *)
+(*   Copyright 2000 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(* Machine-specific command-line options *)
+
+val command_line_options : (string * Arg.spec * string) list
+
+(* Specific operations for the AMD64 processor *)
+
+type addressing_mode =
+    Ibased of string * int              (* symbol + displ *)
```

Comment:
```suggestion
    Ibased of string * sym_global * int              (* symbol + displ *)
```

Reply:
we need to update the [dune](https://github.com/ocaml-flambda/flambda-backend/blob/51d125bc17c8e73e978d47a6439350c6f024436d/backend/dune#L21C2-L21C49) file to copy `ARCH/arch.mli` over to the parent directory.